app/src/main/java/org/go/sopt/HomeFragment.kt

```diff
@@ -0,0 +1,106 @@
+package org.go.sopt
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.example.seminar1.R
+import com.example.seminar1.databinding.FragmentHomeBinding
+import org.go.sopt.Adapter.ViewPagerAdapter
+import org.go.sopt.model.ResponseUser
+import retrofit2.Call
+import retrofit2.Response
+
+// TODO: Rename parameter arguments, choose names that match
+// the fragment initialization parameters, e.g. ARG_ITEM_NUMBER
+private const val ARG_PARAM1 = "param1"
+private const val ARG_PARAM2 = "param2"
+
+/**
+ * A simple [Fragment] subclass.
+ * Use the [HomeFragment.newInstance] factory method to
+ * create an instance of this fragment.
+ */
+class HomeFragment : Fragment() {
+
+    private var _binding: FragmentHomeBinding? = null
+    private val binding : FragmentHomeBinding
+        get() = requireNotNull(_binding) { "앗 ! _binding이 null이다 !" }
+
+    private val reqresService = ReqresServicePool.reqresService
+    private  lateinit var viewPagerAdapter : ViewPagerAdapter
+
+    // TODO: Rename and change types of parameters
+    private var param1: String? = null
+    private var param2: String? = null
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        arguments?.let {
+            param1 = it.getString(ARG_PARAM1)
+            param2 = it.getString(ARG_PARAM2)
+        }
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
```

Comment:
```suggestion
    ): View {
```
상속하는 View가 과연 Nullable일까요?